django/api/admin.py

```diff
@@ -35,6 +35,29 @@ def has_delete_permission(self, request, obj=None):
 @admin.register(GoElectricRebateApplication)
 class GoElectricRebateApplicationAdmin(admin.ModelAdmin):
     search_fields = ["drivers_licence", "id", "status"]
+    exclude = ("sin",)
+    readonly_fields = (
+        "id",
+        "last_name",
+        "first_name",
+        "middle_names",
+        "email",
+        "user_is_bcsc",
+        "address",
+        "city",
+        "postal_code",
+        "drivers_licence",
+        "date_of_birth",
+        "tax_year",
+        "doc1",
```

Comment:
you can get rid of doc_1 and doc_2 since we're using the doc1_tag and doc2_tag

Reply:
also, you probably want to include doc1 and doc2 in the excluded_fields tuple

Reply:
shouldn't the docs be visible in the application?